specification/columns/commitmentdiscountquantity.md

```diff
@@ -0,0 +1,59 @@
+# Commitment Discount Quantity
+
+Commitment Discount Quantity is the amount of a [*commitment discount*](#commitment-discount) purchased or accounted for in *commitment discount* related [*rows*](#glossary:row). The amounts are denominated in [*Commitment Discount Units*](#glossary:commitmentdiscountunit). For a purchase, the Commitment Discount Quantity is either the total of *Commitment Discount Units* over a *commitment discount's* [*term*](#glossary:term) or the apportioned amount for the [*charge period*](#glossary:chargeperiod). For committed usage, the Commitment Discount Quantity is either the number of *Commitment Discount Units* consumed by a [*row*](glossary:#row) that is covered by a *commitment discount* or is the unused portion of a *commitment discount* over a *charge period*. The CommitmentDiscountQuantity column only applies to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount). Commitment Discount Quantity is commonly used in commitment discount analysis and optimization use cases.
+
+For usage-based *commitment discounts* (i.e. when *CommitmentDiscountCategory* is "Usage"), the Commitment Discount Quantity reflects the amount of resource hours consumed. If *size-flexibility* is applicable, this amount is further multiplied by a pre-determined, SKU attribute called an *instance type ratio*, to produce a normalized quantity based on [*resource's*](#glossary:resource) [*instance-size*](#glossary:instance-size). *Size-flexibility* allows a usage-based *commitment discount* to cover less expensive SKUs with smaller *instance type ratios*, or conversely, partially cover a more expensive SKU with a larger *instance type ratio*.
```

Comment:
```suggestion
For usage-based *commitment discounts* (i.e. when *CommitmentDiscountCategory* is "Usage"), the Commitment Discount Quantity reflects the amount of consumed resources in units applicable to the committed units of the resource, such as hours. The unit should align to the committed unit of the purchase of the commitment. 
```

Reply:
I am rejecting this suggestion based on the sample data out further showing size flexibility examples across in commitment discount quantity columns and unused size flex quantities being derivable as compared to consumed quantities and units.